e2e/main_test.go

```diff
@@ -26,7 +26,7 @@ const (
 	splunkDriverName                 = "splunk"
 	containerIDKey                   = "--container-id"
 	containerNameKey                 = "--container-name"
-	testContainerID                  = "test-container-id"
+	testContainerID                  = "210987654321"
```

Comment:
Why the change from `test-container-id`?

Reply:
Fluentd only keeps the first 12 digits so it would be "test-contain" from the original one. We can't match them directly.

I think it is natural to make the testContainerID as real container ID format so didn't leave comment here.

Reply:
cool thanks for the info!